src/Renderers/ErrorPageRenderer.php

```diff
@@ -15,6 +15,21 @@ class ErrorPageRenderer
 {
     public function render(Throwable $throwable): void
     {
+        $vitejsAutoRefresh = '';
+
+        if (class_exists('Illuminate\Foundation\Vite')) {
+
+            $vite = app(\Illuminate\Foundation\Vite::class);
+
+            $hotFile = version_compare(app()->version(), '9.27.0') >= 0
+                             ? $vite->hotFile()
+                             : public_path('/hot');
+
+            if (is_file($hotFile)) {
```

Comment:
Should there be also a check if `vite.config.js` or `vite.config.ts` exist? Some people may have a file named `hot` already, which may have nothing to do with the vite hot reload.

Though I think this is unlikely.